doc/glossary.rst

```diff
@@ -0,0 +1,1216 @@
+        categorical features.
+
+    clone
+        To copy an :term:`estimator instance` and create a new one with
+        identical :term:`parameters`, but without any fitted
+        :term:`attributes`, using :func:`~skelarn.base.clone`.
+
+        When ``fit`` is called, a :term:`meta-estimator` usually clones
+        a wrapped estimator instance before fitting the cloned instance.
+        (Exceptions, for legacy reasons, include
+        :class:`~pipeline.Pipeline` and
+        :class:`~pipeline.FeatureUnion`.)
+
+    common tests
+        This refers to the tests run on almost every estimator class in
+        Scikit-learn to check they comply with basic API conventions.  They are
+        available for external use through
+        :func:`utils.estimator_checks.check_estimator`.
+
+        Note: Some exceptions to the common testing regime are currently
+        hard-coded into the library, but we hope to replace this by marking
+        exceptional behaviours on the estimator using semantic :term:`estimator
+        tags`.
+
+    cross validation splitter
+    CV splitter
+        A non-estimator family of classes used to split a dataset into a
+        sequence of train and test portions (see :ref:`cross_validation`),
+        by providing :term:`split` and :term:`get_n_splits` methods.
+        Note that unlike estimators, these do not have :term:`fit` methods
+        in which to perform parameter validation, etc., and do not provide
+        :term:`set_params` or :term:`get_params`.
+
+    deprecation
+        We use deprecation to slowly violate our :term:`backwards
+        compatibility` assurances, usually to to:
+
+        * change the semantics of the default value of a parameter; or
+        * remove a parameter, attribute, method, class, etc.
+
+        We will ordinarily issue a warning when a deprecated element is used,
+        although there may be limitations to this.  For instance, we will raise
+        a warning when someone sets a parameter that has been deprecated, but
+        may not when they access that parameter's attribute on the estimator
+        instance.
+
+        See the :ref:`Contributors' Guide <contributing_deprecation>`.
+
+    docstring
+        The embedded documentation for a module, class, function, etc., usually
+        in code as a string at the beginning of the object's definition, and
+        accessible as the object's ``__doc__`` attribute.
+
+        We try to adhere to `PEP257
+        <http://www.python.org/dev/peps/pep-0257/>`_, and follow `NumpyDoc
+        conventions <numpydoc.readthedocs.io/en/latest/format.html>`_.
+
+    double underscore notation
+        When specifying parameter names for nested estimators, ``__`` may be
+        used to separate between parent and child in some contexts.
+        See :term:`parameter`.
+
+    dtype
+    data type
+        TODO. Mention casting.
+
+    duck typing
+        We try to apply [duck
```

Comment:
The link [did not rendered](https://15591-843222-gh.circle-artifacts.com/0/home/ubuntu/scikit-learn/doc/_build/html/stable/glossary.html#term-duck-typing), probably because of the new line

Reply:
Or because I wrote it in markdown!

Reply:
indeed :)